packages/editor/src/client/panel/mapLeafletEditPanel.tsx

```diff
@@ -0,0 +1,213 @@
+import React, {useEffect, useState} from 'react'
+import {MaterialIconClose} from '@karma.run/icons'
+import {MarkerPoint} from '../utility'
+import axios from 'axios'
+
+import {
+  NavigationButton,
+  Panel,
+  PanelHeader,
+  PanelSection,
+  TextInput,
+  Spacing,
+  ListValue,
+  ListInput,
+  Box,
+  FieldProps,
+  AutocompleteInput,
+  AutocompleteInputListProps,
+  SelectList,
+  SelectListItem,
+  MarginProps
+} from '@karma.run/ui'
+
+import {MapLeafletItem} from '../blocks/types'
+import nanoid from 'nanoid'
+
+export interface MapLeafletEditPanel {
+  initialItems: ListValue<MapLeafletItem>[]
+  onClose?(items: ListValue<MapLeafletItem>[]): void
+}
+
+export function MapLeafletEditPanel({onClose, initialItems}: MapLeafletEditPanel) {
+  const defaultItemValue = {
+    address: 'Brauerstrasse 42, Zürich',
+    lat: 47.3778762,
+    lng: 8.5271078,
+    title: '',
+    description: '',
+    image: null
+  }
+  const [items, setItems] = useState<ListValue<MapLeafletItem>[]>(() =>
```

Comment:
why the function?

Reply:
We made this workaround to avoid the "empty-chip-problem" that we had when we would initialise a markerpoint. We had a discussion about this and decided to add for the moment a default address (Brauerstrasse) until we switch the UI-framework since the problem is related to the karma.run-UI.